RecoEgamma/EgammaTools/python/egammaObjectModificationsInMiniAOD_cff.py

```diff
@@ -98,10 +98,24 @@ def setup_mva(val_pset,cat_pset,prod_name,mva_name):
 for valueMapName in RecoEgamma.EgammaTools.calibratedPhotonProducer_cfi.calibratedPhotonProducer.valueMapsStored:
     setattr(reducedEgammaEnergyScaleAndSmearingModifier.photon_config,valueMapName,cms.InputTag("reducedEgamma",prefixName("calibPho",valueMapName)))
 
+#############################################################
+# 8X to 9X modifiers (fills in variables new to 9X w.r.t 8X)
+#############################################################
+egamma8XTo9XModifier = cms.PSet(
```

Comment:
what is the meaning of "To9X" in this context?
We should support a working version in 10X as well.
So, the destination can perhaps be dropped